rts/lra-examples/cdi-participant/pom.xml

```diff
@@ -95,22 +95,28 @@
         <dependency>
             <groupId>org.jboss.narayana.rts</groupId>
             <artifactId>narayana-lra</artifactId>
+            <version>${project.version}</version></dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jackson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client-jackson</artifactId>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.jboss.narayana.rts</groupId>            
+            <artifactId>narayana-lra</artifactId>
             <version>${project.version}</version>
         </dependency>
-
-        <!-- MP LRA CDI support defines annotations for controlling the lifecycle
-             of LRAs and for controlling participation in an LRA
-             -->
         <dependency>
-            <groupId>org.eclipse.microprofile.lra</groupId>
-            <artifactId>microprofile-lra-api</artifactId>
-            <version>${version.microprofile.lra}</version>
+          <groupId>org.jboss.narayana.rts</groupId>
+          <artifactId>lra-service-base</artifactId>
+          <version>${project.version}</version>
         </dependency>
 
-       <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy</artifactId>
-        </dependency>
+
```

Comment:
I think this extra line could be removed

Reply:
Thanks @tomjenkinson , I hope I addressed you request in my last commit.